app/services/menu/show.rb

```diff
@@ -0,0 +1,24 @@
+class Menu
```

Comment:
Read, please, article, added above, about facade pattern.
Usually for `show/index/new` pages there is enough to use presenter instead of common service